test/package-json-test.js

```diff
@@ -99,7 +99,7 @@ describe('generating package.json', function() {
         baseURL: '/',
         locationType: 'auto',
         EmberENV: { FEATURES: {} },
-        APP: { name: 'module-whitelist', version: '0.0.0+', autoboot: false },
+        APP: { name: 'module-whitelist', version: '0.0.0', autoboot: false },
```

Comment:
@rwjblue This was the only failing test, but I have no clue why there was a plus sign in this version number.

Reply:
Yeah the current master is red due to these tests failing. Thanks for fixing this.